src/core/execution/SAMLauncherExecution.ts

```diff
@@ -20,7 +20,7 @@ export class SAMLauncherExecution implements Execution {
 
   private target: Unit = null;
 
-  private searchRangeRadius = 75;
+  private searchRangeRadius = 100;
```

Comment:
While I think increasing the size of SAM range is a good idea long term, until we implement building density restrictions and/or a hotkey to launch a bombs (for  successful saturation strikes), hitting a cluster of SAMs with the outer edge an H-Bomb is currently the only viable counter to islands with a fleet of warships and 5 or 6 or more SAMs.

the inner destruction radius on the H-bombs is 80, and the outer is 100, suggest increasing SAM range only marginally until the other changes mentioned above are implemented.

```suggestion
  private searchRangeRadius = 80;
```